components/03-modules/range/range.js

```diff
@@ -10,19 +10,16 @@
   }
 
   function createRangeSlider(rangeFilter) {
-
-    const rangeSlider = rangeFilter.querySelector('.range-filter__slider'),
-          inputMax = rangeFilter.querySelector('.range-filter__field--upper'),
-          inputMin = rangeFilter.querySelector('.range-filter__field--lower');
+    const inputMax = rangeFilter.querySelector('.range-filter__field--upper'),
+          inputMin = rangeFilter.querySelector('.range-filter__field--lower'),
+          rangeSlider = rangeFilter.querySelector('.range-filter__slider');
 
     rangeSlider.classList.add('noUi-extended');
     initNoUiSlider(rangeSlider);
     bindEvents(rangeSlider, inputMin, inputMax);
-
   }
 
   function initNoUiSlider(rangeSlider) {
-
     noUiSlider.create(rangeSlider, {
       start: [0, 1000],
       connect: true,
```

Comment:
you are using tooltips as true and hidding it in the preview. Maybe delete config with this?

Reply:
And the question is we can pass as parameter this values. Like range and start?